ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/recovery/transactions/TransactionCacheItem.java

```diff
@@ -86,7 +86,15 @@ private boolean loadTransaction(boolean firstLoad)
 		} 
 	    }
 	}
-	else if (_type.equals(ServerTransaction.typeName()))
+    else if (_type.equals(ServerTransaction.typeName() + "/JCA"))
+    {
+        try {
+            _transaction = (RecoveringTransaction) Class.forName("com.arjuna.ats.internal.jta.recovery.jts.jca.coordinator.RecoveredServerTransaction").getConstructor(new Class[] {Uid.class, String.class}).newInstance(new Object[] {_uid, _type});
```

Comment:
This hardcoded classname arose because you copied RecoveredServerTransaction from jts to jtax. How different it the jtax version you created - I have just gone back and I see the new version is the same as teh old one.